Core/Services/StitchCoreRemoteMongoDBService/Sources/StitchCoreRemoteMongoDBService/Sync/CoreDocumentSynchronizationConfig.swift

```diff
@@ -299,7 +299,8 @@ internal struct CoreDocumentSynchronization: Hashable {
     }
 
     internal static func filter(forNamespace namespace: MongoNamespace) -> Document {
-        return [CoreDocumentSynchronization.Config.CodingKeys.namespace.rawValue: namespace.description]
+        return [CoreDocumentSynchronization.Config.CodingKeys.namespace.rawValue:
+            try! BSONEncoder().encode(namespace)]
```

Comment:
Hmm... IMO, we should disable the linter on lines for these types of scenarios. Why not?

Reply:
that still runs the risk of failing if the implementation of BSONEncoder().encode() changes in the future. We've seen with recent Go driver changes on the Stitch server that stuff like this can randomly change.

That being said, I think it's fine to leave as is for now, but let's discuss again when we get to the ticket that lints this stuff.